src/runtime/stores/formatters.ts

```diff
@@ -57,6 +57,14 @@ const formatMessage: MessageFormatter = (id, options = {}) => {
     }
 
     message = defaultValue ?? id;
+
+    if (getOptions().onMissingMessageHandler) {
+      getOptions().onMissingMessageHandler(
+        getPossibleLocales(locale),
```

Comment:
I think we should pass the current locale only

```suggestion
        locale,
```